test/integration/default/serverspec/kafka_spec.rb

```diff
@@ -118,22 +101,20 @@
     # The consumer command allows a user to 'listen' for messages on the topic and write them to STDOUT as they come in
     # In this case we are re-directing STDOUT to a file so we can read later
     # We also run this as a background process so we can also start the producer
-    Kernel.system "/opt/kafka/bin/kafka-console-consumer.sh --zookeeper localhost:2181 --whitelist #{topic} >> /tmp/consumer.out 2>&1 &"
+    Kernel.system "/opt/kafka/bin/kafka-verifiable-consumer.sh --broker-list localhost:9092 --topic #{topic} --group-id #{group} >> /tmp/consumer.out 2>&1 &"
+
+    # Give consumer 5s to start up
+    sleep 5
 
     # The producer is a command that allows a user to write input to the console as 'messages' to the topic, separated by new line characters
     # In this case we run the command and write the same message several times over 5s in an attempt to ensure the consumer saw the message
-    IO.popen("/opt/kafka/bin/kafka-console-producer.sh --topic #{topic} --broker-list localhost:6667 2> /dev/null", mode='r+') do |io|
```

Comment:
This was hanging for me. Switched to these new commands which seemed simpler/easier to use